docs/pages/blog/mui-x-v7-beta.md

```diff
@@ -0,0 +1,225 @@
+Most breaking changes are in place, and we shift our focus towards refining and expanding over the new version.
+
+## Table of contents
+
+- [Data Grid](#data-grid)
+  - [Sticky headers](#sticky-headers)
+  - [Improved columns panel design](#improved-columns-panel-design)
+  - [Date Object support in filter model](#date-object-support-in-filter-model)
+  - [New stable features](#new-stable-features)
+  - [Smaller bundle size](#smaller-bundle-size)
+- [Tree View](#tree-view)
+  - [RichTreeView](#richtreeview-new-component)
+- [Charts](#charts)
+  - [Reference Line](#reference-line)
+- [Date Pickers](#date-pickers)
+  - [Support for date-fns v3](#support-for-date-fns-v3)
+  - [Date Time Range Picker](#date-time-range-picker)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [What's next](#whats-next)
+- [Migration guide](#migration-guide)
+- [How to get involved](#how-to-get-involved)
+
+## Data Grid
+
+We have introduced critical breaking changes designed not only to enhance the developer experience but also to bring the user experience to a new level.
+
+### Sticky headers
+
+The column headers and pinning implementations have been refactored for an improved user experience.
+You'll notice a more responsive Data Grid with smoother horizontal scrolling and better screen reader support.
+The video below showcases some of these improvements.
+
+<video preload="metadata" autoplay muted loop playsinline width="690" height="417" controls>
+  <source src="/static/blog/mui-x-v7-beta/sticky-headers.mp4" type="video/mp4">
+</video>
+
+:::warning
+The new experience is far better, but the solution has a limitation: **column headers can no longer be transparent** because content scrolls behind them.
+
+If you're using our theming system, the DataGrid will adapt to your application seamlessly.
+
+Without the theming system, you may need to manually set an appropriate background color to maintain your design.
+:::
+
+### Improved columns panel design
+
+The column management panel now features a sleeker, checkbox-based design, replacing the previous toggle button style. Additionally, we've extracted the internal component to facilitate the introduction of the upcoming pivoting UI.
+
+<img width="310" alt="new column management panel design" src="/static/blog/mui-x-v7-beta/column-management-panel.png">
+
+### Date Object support in filter model
+
+The `filterModel` now supports `Date` objects for `date` and `dateTime` column types, providing a more intuitive and efficient filtering experience.
+
+While string values remain compatible for these types, any updates to the `filterModel` made through the UI (such as via the filter panel) will now automatically use `Date` objects, ensuring consistency and ease of data handling.
+
+### Smaller bundle size
+
+The introduction of a separate entry point for locales has significantly optimized our bundle size.
```

Comment:
This is misleading otherwise, the reality is that the bundle size has **increased** for the use cases that matter.

```suggestion
The introduction of a separate entry point for locales has significantly optimized the bundle size of the barrel index in development mode.
```